File: monitor/alerts/instance-get-example/instance-get-example.5.x.php
<?php
// Get the PHP helper library from https://twilio.com/docs/libraries/php
require_once '/path/to/vendor/autoload.php'; // Loads the library
use Twilio\Rest\Client;

// Your Account Sid and Auth Token from twilio.com/user/account
$sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX";
$token = "your_auth_token";

$client = new Client($sid, $token);

// Get an object from its sid. If you do not have a sid,
// check out the list resource examples on this page
$alert = $client->monitor
    ->alerts("NO5a7a84730f529f0a76b3e30c01315d1a")
    ->fetch();

echo $alert->alertText;

File: monitor/alerts/instance-get-example/instance-get-example.5.x.py
# Download the Python helper library from twilio.com/docs/python/install
from twilio.rest.monitor import TwilioMonitorClient

# Your Account Sid and Auth Token from twilio.com/user/account
account_sid = "ACXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXXX"
auth_token = "your_auth_token"
client = TwilioMonitorClient(account_sid, auth_token)

alert = client.alerts.get("NO5a7a84730f529f0a76b3e30c01315d1a")
print(alert.alert_text)
